src/persist/src/operators/input.rs

```diff
@@ -113,17 +113,18 @@ mod tests {
     use timely::dataflow::operators::Capture;
 
     use crate::error::Error;
-    use crate::mem::MemPersister;
+    use crate::file::FilePersister;
     use crate::persister::Persister;
     use crate::Id;
 
     use super::*;
 
     #[test]
     fn new_persistent_unordered_input() -> Result<(), Error> {
-        let mut p = MemPersister::new();
+        let mut p = FilePersister::new("new_persistent_unordered_input/");
```

Comment:
keep this test as the mempersister since we don't have to worry about cleaning up test state with it